myst_nbs/generalized_linear_models/GLM-robust.myst.md

```diff
@@ -15,16 +15,15 @@ kernelspec:
 
 This tutorial first appeard as a post in small series on Bayesian GLMs on:
 
-  1. [The Inference Button: Bayesian GLMs made easy with PyMC3](http://twiecki.github.com/blog/2013/08/12/bayesian-glms-1/)
-  2. [This world is far from Normal(ly distributed): Robust Regression in PyMC3](http://twiecki.github.io/blog/2013/08/27/bayesian-glms-2/)
-  3. [The Best Of Both Worlds: Hierarchical Linear Regression in PyMC3](http://twiecki.github.io/blog/2014/03/17/bayesian-glms-3/)
+  1. [The Inference Button: Bayesian GLMs made easy with PyMC](http://twiecki.github.com/blog/2013/08/12/bayesian-glms-1/)
+  2. [This world is far from Normal(ly distributed): Robust Regression in PyMC](http://twiecki.github.io/blog/2013/08/27/bayesian-glms-2/)
+  3. [The Best Of Both Worlds: Hierarchical Linear Regression in PyMC](http://twiecki.github.io/blog/2014/03/17/bayesian-glms-3/)
```

Comment:
Do some of these notebooks have also been adapted to examples here? If so I think it would be better to link to the updated version that will use v4 (with always each of the notebooks awknowledging they were adapted and linking to the original notebook)

Reply:
Yeah those haven't been ported.